internal/api/impl/auth/service.go

```diff
@@ -53,7 +54,7 @@ func saveProviderInfo(old v1.UserSpec, uInfoProvider v1.OAuthProvider) (v1.UserS
 	foundPerfectMatch := false
 	for _, provider := range old.OauthProviders {
 		if provider != uInfoProvider {
-			return old, false, errors.New("this user is already registered with a different oauth provider context")
+			return old, false, fmt.Errorf("this user is already registered with a different oauth provider context (%s)", provider)
```

Comment:
I'm realizing I forgot provider.Name

Reply:
I'm using provider.Issuer as we don't save the provider name in the user. This will be the public URL of the provider. The one the user is redirected to when login so nothing secret.